ios/RNSScreenStackHeaderConfig.h

```diff
@@ -50,7 +60,8 @@ typedef NS_ENUM(NSInteger, RNSScreenStackHeaderSubviewType) {
 
 + (RNSScreenStackHeaderSubviewType)RNSScreenStackHeaderSubviewType:(id)json;
 + (UIBlurEffectStyle)UIBlurEffectStyle:(id)json;
-+ (UISemanticContentAttribute)RNSScreenDirection:(id)json;
++ (UISemanticContentAttribute)UISemanticContentAttribute:(id)json;
```

Comment:
I changed it beacuse the method name was a relict of some of the previous implementations and could be misleading.